### PR TITLE
Add Stormglass API utils

### PR DESCRIPTION
**Pull request by PysarenkoDS-BWT**
Function  stormGlassGetWeather have next arg object {
  latitude,  // required
  longitude,   // required
  params,   // required
  source,
  start,
  end,
}

For work we need provide `latitude` and `longitude`, I think here we not have questions
then we need provide `params` it's array of param, where we indicate what data we want to receive in the response (I type full data variant in typescript for easy usage) list of variants we can see [here](https://docs.stormglass.io/#/weather?id=point-response)

Next props are not required:
`source` - here we provided source or source[] from which sources we want to see results list [here](https://docs.stormglass.io/#/sources). as defaults return from all available sources
`start` - date of start data
`end` - date of end data

This function has retuned data in next format 
```
[
  {
    "airTemperature": {
      "timestamp": "2022-05-26T00:00:00+00:00",
      "value": 8.275
    },
    "waveHeight": {
      "timestamp": "2022-05-26T00:00:00+00:00",
      "value": 0.4242857142857143
    }
  },
  {
    "airTemperature": {
      "timestamp": "2022-05-26T01:00:00+00:00",
      "value": 7.91
    },
    "waveHeight": {
      "timestamp": "2022-05-26T01:00:00+00:00",
      "value": 0.42142857142857143
    }
  },
]
```

Where list of keys depends on provided `params` and value it's average value from all provided sources

### CODE REVIEW

**Comment by ericboucher** [comment]
Nice start @PysarenkoDS-BWT, a few things:
- let's add a description to your PR
- similarly to `sofar.ts`, we need our utility functions to return values that are formatted in a way that our API and frontend can understand. For example, data needs to be ready to be added to our forecast table. You can have a quick look at https://github.com/aqualinkorg/aqualink-app/pull/729

File: packages/api/src/utils/storm-glass.const.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// dotenv is a dev dependency, so conditionally import it (don't need it in Prod).
+try {
+  // eslint-disable-next-line import/no-extraneous-dependencies, global-require
+  require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (PysarenkoDS-BWT) [reply packages/api/src/utils/storm-glass.const.ts:4]: I agree, but I see some code here `packages/api/src/utils/constants.ts`
And i reused it, may be I don't know  some think  about working this app and I decided that it is necessary

I'm go away from it in next commit

File: packages/api/src/utils/storm-glass.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import axios from 'axios';
+import axiosRetry from 'axios-retry';
+import { STORM_GLASS_API_KEY, STORM_GLASS_BASE_URL } from './storm-glass.const';
+import {
+  StormGlassWeatherProps,
+  StormGlassWeatherQueryProps,
+} from './storm-glass.types';
+
+axiosRetry(axios, { retries: 3 });
```
[the remark this answers is not in the document]
Comment (PysarenkoDS-BWT) [reply packages/api/src/utils/storm-glass.ts:9]:
```
import axios from 'axios';
import axiosRetry from 'axios-retry';

axiosRetry(axios, { retries: 3 });

export axios;
```

Josh G
You mean create some file like that and use axios from this file?

File: packages/api/src/utils/storm-glass.const.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const STORM_GLASS_BASE_URL = 'https://api.stormglass.io/v2';
```
Comment (ericboucher) [inline packages/api/src/utils/storm-glass.const.ts:1]: let's move this to `constants.ts`

**Review by ericboucher — APPROVED** [review]

**Pull request merged**